src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -1501,6 +1501,26 @@ public async Task BaseList2()
             await VerifyItemExistsAsync(AddUsingDirectives("using System;", @"class CL : B, $$"), @"System");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task BaseList_Class_PartiallyWritten()
+        {
+            await VerifyItemExistsAsync(@"
+class BaseClass {}
+
+class C : Base$$
+", "BaseClass");
+        }
```

Comment:
can you verify this as well if you have:

```
class C : Base$$, IFoo
{
}
```

?

thanks!

Reply:
There is already one:
```cs
[Fact, Trait(Traits.Feature, Traits.Features.Completion)]
public async Task BaseList_Class_BeforeOtherItems_PartiallyWritten()
{
    await VerifyItemExistsAsync(@"
class BaseClass {}
interface I {}

class B : Base$$, I
", "BaseClass");
    }
```
I don't think that difference in `{}` is worth creating another test here

Reply:
Is there a test for `class B : $$, I`. ?

Reply:
Yes:
```cs
[Fact, Trait(Traits.Feature, Traits.Features.Completion)]
public async Task BaseList_Class_BeforeOtherItems()
{
    await VerifyItemExistsAsync(@"
class Base {}
interface I {}

class B : $$, I
", "Base");
}
```